app/views/landing_page/_cards.html.erb

```diff
@@ -1,7 +1,7 @@
 <div class="row card-container">
   <div class="col-lg-4 my-4 my-lg-0">
     <div class="card h-100">
-      <%= image_tag 'locations.png', alt: '', class: 'card-img-top' %>
+      <%= image_tag 'locations.png', alt: 'Image of library interior', class: 'card-img-top' %>
```

Comment:
Genuinely curious where we stand on alt text for things like this. I intentionally left these blank per:

https://html.spec.whatwg.org/multipage/images.html#a-purely-decorative-image-that-doesn't-add-any-information

Reply:
It just seemed sad to include the attribute with a blank string!

Reply:
I certainly don't mind it either way, I just don't want to keep leaving them blank if that's not something we do around here.

Reply:
I can remove them for now. I see the spec you linked to. We could discuss more generally at our next standup.

Reply:
@taylor-steve removed.